guides/common/modules/proc_configuring-rhui-alternate-content-sources.adoc

```diff
@@ -0,0 +1,18 @@
+[id="Configuring_RHUI_Alternate_Content_Sources_{context}"]
+= Configuring RHUI Alternate Content Sources
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Alternate Content Sources*.
+. Click *Add Source* and set the *Source type* as *RHUI*.
+. Generate RHUI certificates using the command provided in the {ProjectWebUI}.
+Ensure that you pass the repo labels of the desired repositories.
+. In the *Name* field, enter a name for the alternate content source.
+. Optional: In the the *Description* field, provide a description for the ACS.
+. Select {SmartProxies} to which the alternate content source is to be synced.
+. Optional: Select `Use HTTP proxies` if you want the ACS to use the {SmartProxy}'s HTTP proxy.
+. Enter the Base URL of the Red Hat Update Infrastructure server.
+. Enter a comma-separated list of Subpaths.
+. Provide the *Content Credentials*, if these are needed.
+. If SSL verification is required, enable *Verify SSL* and select the SSL CA certificate.
+. Review details and click *Add*.
+. Navigate to *Content* > *Alternate Content Sources* > click the vertical ellipsis next to the newly created alternate content source > Select *Refresh*.
```

Comment:
Do we need to add the 'CLI Procedure' also here?

Reply:
@vsedmik @ianballou  done!